src/main/resources/assets/terrestria/lang/zh_cn.json

```diff
@@ -1,225 +1,242 @@
+  "block.terrestria.basalt": "玄武岩",
+  "block.terrestria.basalt_slab": "玄武岩台阶",
+  "block.terrestria.basalt_stairs": "玄武岩楼梯",
+  "block.terrestria.basalt_wall": "玄武岩墙",
+  "block.terrestria.smooth_basalt": "平滑玄武岩",
+  "block.terrestria.smooth_basalt_slab": "平滑玄武岩台阶",
+  "block.terrestria.smooth_basalt_stairs": "平滑玄武岩楼梯",
+  "block.terrestria.smooth_basalt_wall": "平滑玄武岩墙",
+  "block.terrestria.basalt_cobblestone": "玄武岩圆石",
+  "block.terrestria.basalt_cobblestone_slab": "玄武岩圆石台阶",
+  "block.terrestria.basalt_cobblestone_stairs": "玄武岩圆石楼梯",
+  "block.terrestria.basalt_cobblestone_wall": "玄武岩圆石墙",
+  "block.terrestria.mossy_basalt_cobblestone": "生苔的玄武岩圆石",
+  "block.terrestria.mossy_basalt_cobblestone_slab": "生苔的玄武岩圆石台阶",
+  "block.terrestria.mossy_basalt_cobblestone_stairs": "生苔的玄武岩圆石楼梯",
+  "block.terrestria.mossy_basalt_cobblestone_wall": "生苔的玄武岩圆石墙",
+  "block.terrestria.basalt_bricks": "玄武岩石砖",
+  "block.terrestria.basalt_brick_slab": "玄武岩石砖台阶",
+  "block.terrestria.basalt_brick_stairs": "玄武岩石砖楼梯",
+  "block.terrestria.basalt_brick_wall": "玄武岩石砖墙",
+  "block.terrestria.mossy_basalt_bricks": "生苔的玄武岩石砖",
+  "block.terrestria.mossy_basalt_brick_slab": "生苔的玄武岩石砖台阶",
+  "block.terrestria.mossy_basalt_brick_stairs": "生苔的玄武岩石砖楼梯",
+  "block.terrestria.mossy_basalt_brick_wall": "生苔的玄武岩石砖墙",
+  "block.terrestria.basalt_button": "玄武岩按钮",
+  "block.terrestria.basalt_pressure_plate": "玄武岩压力板",
+  "block.terrestria.chiseled_basalt_bricks": "錾制玄武岩石砖",
+  "block.terrestria.cracked_basalt_bricks": "裂玄武岩石砖",
+  "block.terrestria.basalt_grass_block": "火山灰土草方块",
+  "block.terrestria.basalt_grass_path": "火山灰土草径",
+  "block.terrestria.basalt_sand": "玄武岩砂",
+  "block.terrestria.basalt_dirt": "火山灰土",
+  "block.terrestria.basalt_podzol": "灰化火山灰土",
+  "block.terrestria.indian_paintbrush": "印地安画笔花",
+  "block.terrestria.monsteras": "龟背竹",
+  "block.terrestria.potted_indian_paintbrush": "印地安画笔花盆栽",
+  "block.terrestria.potted_monsteras": "龟背竹",
+  "block.terrestria.japanese_maple_shrub_sapling": "日本枫木灌木树苗",
+  "block.terrestria.japanese_maple_shrub_leaves": "日本枫木灌木树叶",
+  "block.terrestria.potted_redwood_sapling": "红杉木树苗盆栽",
+  "block.terrestria.potted_hemlock_sapling": "铁杉木树苗盆栽",
+  "block.terrestria.potted_rubber_sapling": "橡胶木树苗盆栽",
+  "block.terrestria.potted_cypress_sapling": "柏木树苗盆栽",
+  "block.terrestria.potted_willow_sapling": "柳木树苗盆栽",
+  "block.terrestria.potted_japanese_maple_sapling": "日本枫木树苗盆栽",
+  "block.terrestria.potted_dark_japanese_maple_sapling": "日本红枫木树苗盆栽",
+  "block.terrestria.potted_rainbow_eucalyptus_sapling": "彩虹桉木树苗盆栽",
+  "block.terrestria.potted_sakura_sapling": "樱花木树苗盆栽",
+  "block.terrestria.potted_jungle_palm_sapling": "丛林棕榈树苗盆栽",
+  "block.terrestria.potted_japanese_maple_shrub_sapling": "日本枫木灌木树苗",
+  "item.terrestria.log_turner": "木头旋转器",
+  "item.terrestria.log_turner.tooltip": "对着一个原木使用这个工具,\n 你所对准的面将成为原木的年轮所朝向的面\n 当你所对着的是一个木方块时\n 它将会以木块的年轮以90°的方式旋转着这个方块\n 直到你想让这个木块旋转到你认为的方向时\n 所有的旋转将会以顺时针进行\n 当你按住潜行进行旋转木块时\n 木块不会以表面的年轮进行旋转",
```

Comment:
"Use on a log to instantly set\n its axis to the side you clicked\n When used on a quarter log, it\n will try to set the axis, and if\n it already matches, it will then\n rotate the bark side clockwise\n When used while sneaking, it will\n cycle the axis and bark side",

用在原木時會使原木朝向用了工具的面\n如果已經是一樣的面, 就會使橡皮面轉90°順時針\n如果潜行時利用道具就會在橡皮面與軸而循環.

Reply:
你所説的一樣的面 其實指的是柳木 紅木 等單一等木頭方塊
要不改成這樣 

对着一个原木使用这个工具,\n 你所对准的面将成为原木朝向你使用此工具的面\n 当你所对着的是一个木方块时\n 如果你对着的是一个木方块,它就会使橡皮面转90°顺时针方向旋转\n 直到你想让这个木块旋转到你想要到达的方向\n 所有的转向都会以顺时针进行\n 当你按住潜行进行旋转木块时\n 木块不会以表面的年轮进行旋转

Reply:
OK, 只是怕會不會太長?

Reply:
沒事的 有些模組長句子比這個還多幾倍呢